include/grpc/impl/codegen/port_platform.h

```diff
@@ -196,7 +196,7 @@
 #define GPR_GCC_TLS 1
 #define GPR_LINUX 1
 #define GPR_LINUX_LOG
-#define GPR_LINUX_MULTIPOLL_WITH_EPOLL 1
+#define GPR_POSIX_MULTIPOLL_WITH_POLL 1
```

Comment:
I think this changes a constant that is currently used to one that is not currently used.

Also, why are you changing the `__linux__` section of `port_platform.h` for a change related to NACL?